defaults/main.yml

```diff
@@ -1,17 +1,17 @@
 ---
-openio_filebeat_version: 6.7.0
-
-openio_filebeat_namespace: OPENIO
-openio_filebeat_gridinit_dir: "/etc/gridinit.d/{{ openio_filebeat_namespace }}"
-openio_filebeat_gridinit_file_prefix: ""
-openio_filebeat_pid_directory:
-  "/run/filebeat/{{ openio_filebeat_namespace }}/filebeat-{{ openio_filebeat_serviceid }}"
+openio_filebeat_version: 7.5.0
 
 openio_filebeat_serviceid: "0"
-openio_filebeat_volume: "/var/lib/oio/sds/{{ openio_filebeat_namespace }}/filebeat-{{ openio_filebeat_serviceid }}"
+openio_filebeat_namespace: OPENIO
+openio_filebeat_gridinit_dir: "/etc/gridinit.d"
+openio_filebeat_gridinit_file_prefix: "{{ openio_filebeat_namespace }}-"
+openio_filebeat_pid_dir: "/run/oio/sds/{{ openio_filebeat_namespace }}/{{ openio_filebeat_servicename }}.pid"
```

Comment:
"/run/oio/sds/{{ openio_filebeat_namespace }}"

Reply:
it's a var, not a default

Reply:
fixed